lib/matplotlib/backends/backend_qt4agg.py

```diff
@@ -54,13 +54,7 @@ def new_figure_manager_given_figure(num, figure):
     return FigureManagerQT(canvas, num)
 
 
-class FigureCanvasQTAggBase(_FigureCanvasQTAggBase):
-    def __init__(self, figure):
-        self._agg_draw_pending = False
-
-
-class FigureCanvasQTAgg(FigureCanvasQTAggBase,
-                        FigureCanvasQT, FigureCanvasAgg):
+class FigureCanvasQTAgg(FigureCanvasQTAggBase, FigureCanvasQT):
```

Comment:
Actually this MRO chain will get even further simplified in #8771 (to `class FigureCanvasQTAgg(FigureCanvasAgg, FigureCanvasQT)`, which seems as simple as one can do :-)), sorry for modifying the same thing twice.